### PR TITLE
Seata-Spring-Boot-Starter-Samples

### PR DESCRIPTION
**Pull request by lkxsoul**
Seata-Spring-Boot-Starter-Samples

### CODE REVIEW

**Comment by jsbxyyx** [comment]
please relation github email and username to git user.email, user.name.

File: seata-spring-boot-starter-samples/account-starter/src/main/java/io/seata/starter/samples/service/AccountService.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    @GetMapping("updateAccount/{sid}/{count}")
+    public boolean updateAccount(@PathVariable String sid, @PathVariable int count) {
+        // 当前库存
+        int whenCount = accountMapper.selectCount(sid);
+        if(count > whenCount){
+            throw new RuntimeException("库存不足，抛出异常...");
+        }
+        int i = accountMapper.updateAccount(sid, count);
+        return i>0?true:false;
```
Comment (xingfudeshi) [inline seata-spring-boot-starter-samples/account-starter/src/main/java/io/seata/starter/samples/service/AccountService.java]: Directly return i>0

Comment (lkxsoul) [reply seata-spring-boot-starter-samples/account-starter/src/main/java/io/seata/starter/samples/service/AccountService.java]: > Directly return i>0
Ok, it's been modified

File: seata-spring-boot-starter-samples/account-starter/src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+server:
+  port: 8999
+spring:
+  application:
+    name: account-starter-samples
+  datasource:
+    driver-class-name: com.mysql.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/seata_server2?useAffectedRows=true&characterEncoding=UTF-8&useSSL=false&serverTimezone=GMT
+    name: root
```
Comment (xingfudeshi) [inline seata-spring-boot-starter-samples/account-starter/src/main/resources/application.yml]: username

File: seata-spring-boot-starter-samples/business-starter/src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,111 @@
+server:
+  port: 8888
+spring:
+  application:
+    name: business-starter-samples
+  datasource:
+    driver-class-name: com.mysql.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/seata_server1?useAffectedRows=true&characterEncoding=UTF-8&useSSL=false&serverTimezone=GMT
+    name: root
```
Comment (xingfudeshi) [inline seata-spring-boot-starter-samples/business-starter/src/main/resources/application.yml]: username

**Comment by slievrly** [comment]
@lkxsoul please  resolve conflicts.

**Pull request closed without merging**